yjit/src/stats.rs

```diff
@@ -355,11 +355,13 @@ make_counters! {
     expandarray_not_array,
     expandarray_rhs_too_small,
 
-    gbpp_wb_required,
-    gbpp_not_gc_guarded,
+    gbp_wb_required,
```

Comment:
Assuming this is a typo

Reply:
Nah, `gbp_` is for `getblockpraram`, different from `gbpp_` for `getblockparamproxy` I got confused and renamed this counter and I'm reverting that here.